### PR TITLE
Omega steps bug

### PR DESCRIPTION
**Pull request by bnmajor**
This fixes issues 2 and 3 mentioned [here](https://github.com/HEXRD/hexrdgui/issues/722#issuecomment-791049713) in #722

### CODE REVIEW

**Comment by bnmajor** [comment]
PEP8 issues are all walrus operator related

File: hexrd/ui/image_load_manager.py [hunk 1]
```diff
@@ -177,9 +177,13 @@ def process_ims(self, postprocess, update_progress):
...
             if len(self.files[0]) > 1:
                 for i, det in enumerate(det_names):
-                    dirs = self.parent_dir
-
-                    ims = ImageFileManager().open_directory(dirs, self.files[i])
+                    dirs = os.path.dirname(self.files[i][0])
+                    options = {
+                        'empty-frames': self.data.get('empty_frames', 0),
+                        'max-file-frames': self.data.get('max_frame_file', 0),
+                        'max-total-frames': self.data.get('max_frames', 0)
```
Comment (joelvbernier) [inline hexrd/ui/image_load_manager.py:184]: We should probably also change the name of the assoc. widgets in the UI

File: hexrd/ui/image_load_manager.py [hunk 2]
```diff
@@ -319,33 +323,23 @@ def add_omega_metadata(self, ims_dict):
         # Add on the omega metadata if there is any
         files = self.data['yml_files'] if 'yml_files' in self.data else self.files
         for key in ims_dict.keys():
+            nframes = self.data['total_frames'][0] * len(files[0])
```
Comment (joelvbernier) [inline hexrd/ui/image_load_manager.py:326]: Do we have the raw imageseries loaded at this point?  If so, `nframes = len(raw_ims)`

Comment (joelvbernier) [inline hexrd/ui/image_load_manager.py:326]: I guess I don't fully understand this

**Review by joelvbernier — APPROVED** [review]

**Pull request merged**